healthmap/mriids2
# Fetch cases.json, deaths.json, and projections.json data files from API

This JIRA addresses the issue - Fetch cases.json, deaths.json, and projections.json data files from API
Fixes - http://jira/browse/MRIID-75

Testing Guidelines - 
- Pull down the branch.
- Update the .env to add `REACT_APP_BACKEND_URL=https://gtwn1uelqb.execute-api.us-east-1.amazonaws.com/dev/api/`
- Start the server
- Check the network tab and make sure that JSON data is returned by API call (as oppose to reading local JSON files)
- Click around the entire UI and make sure its functional (especially when Covid 19 dropdown and its sub-elements are selected)
